completions/cryptsetup

```diff
@@ -11,6 +11,50 @@ _cryptsetup_device()
     _filedir
 }
 
+_cryptsetup_action()
+{
+    local ret IFS=$' \t\n'
+    _comp_dequote "${1-}" || return 1
+    local cmd=${ret:-cryptsetup}
+    local actions=$(
+        {
+            LC_ALL=C "$cmd" --help 2>&1 |
+                sed -n '/^<action> is one of:/,/^[^[:space:]]/s/^[[:space:]]\{1,\}\([^[:space:]]\{1,\}\).*/\1/p'
+            LC_ALL=C man cryptsetup 2>&1 |
+                awk '/^[[:space:]]+[[:alnum:]_]+([[:space:]]+(-[^[:space:].]+|<[^<>]+>|\[[^][]+\]|or))*$/ {print $1}'
+        } | sort -u
+    )
+
+    if [[ ! $actions ]]; then
+        # The fallback action list is extracted from the following source:
```

Comment:
I think we should drop the fallback altogether, unless we know there is an old version we want to support but cannot because its output is not feasible to parse. If there is, we should document what exactly it is.

Reply:
I was just so lazy that I didn't want to check the behavior of old versions. Do we need to check the behavior of old versions of cryptsetup?

Reply:
That's kind of my point -- if we do check old versions and want to support them and can't find a way we like to make that happen otherwise, then I'm fine with a hardcoded list and the strings + grep approach. My opinion is that if we go to such lengths, we should document the reasoning and affected versions _why_ we are doing it, why it is important here etc.

Without verifying from the sources, I believe our current standard is not to use fallback lists at all (we do have them in a few places, but it's not a standard), and the strings+grep verification is completely new, so I'm just validating if there's something exceptionally important here that warrants that stuff.